var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -331,6 +337,11 @@ def determine_bootstrap_options(self, spec, with_libs, options):
         if '+python' in spec:
             options.append('--with-python=%s' % spec['python'].command.path)
 
+        if '~icu' in spec:
```

Comment:
Curious.  Why the inconsistency here versus line 380?

Reply:
Ah, I only discovered later that `--with-icu` was allowed, it's not in the docs. Let me make it consistent